script/autoapi/autoapi

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python
+
+""" Automatically generate API headers from a specification of the JSON
+    messsages that are part of Measurement Kit's FFI API. """
+
+import jinja2
+import json
+import re
+import subprocess
+import sys
+
+class Attribute(object):
+    """ Attribute of a class. """
+
+    def __init__(self, cxx_type, key, value=None):
+        self.cxx_type = cxx_type
+        self.key = key
+        self.value = value if value is not None else "{}"
+        # Casting from/to JSON is required for booleans because the FFI
+        # implementation uses integers instead of booleans.
```

Comment:
Note: this cannot be changed without heavy internals refactoring. So, it can be done, but now we're working on the external API, so it's quite not the time to start refactoring the internals.